Gemfile

```diff
@@ -17,3 +17,4 @@ gem 'json', '~> 1.8.2'
 gem 'git', '~> 1.2.6'
 gem 'trollop', '~> 2.0'
 gem 'coderay', '~> 1.1.0'
+gem 'rb-inotify', '~> 0.9'
```

Comment:
I checked and it's not necessary to include ```rb-inotify``` as it is already being included in the dependency chain:
```awestruct-0.5.7``` depends on ```listen-3.1.5``` which depends on ```rb-inotify (>= 0.9.7, ~> 0.9)```